src/server/capacity_unit_calculator.cpp

```diff
@@ -97,6 +105,8 @@ void capacity_unit_calculator::add_get_cu(int32_t status,
         return;
     }
 
+    _read_hotkey_collector->capture_raw_key(key, key.size() + value.size());
+
```

Comment:
I think it's not needed to add capture_raw_key in this PR, since only this is just a 'fundamental framework' PR, and some othe read/write paths like add_write_cu missing call it.

Reply:
`capture_raw_key` is a fundamental interface of `hotkey_collector`, I want to show how to use this function